tests/tst_calendarevent/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(SRC tst_calendarevent.cpp)
+
+add_executable(tst_calendarevent ${SRC})
+
+target_include_directories(tst_calendarevent PRIVATE
+	${PROJECT_SOURCE_DIR}/src
+	${TARGET_MOC_DIR})
+
+target_link_libraries(tst_calendarevent
+	Qt5::Qml
+	Qt5::Test
+	KF5::CalendarCore
+	nemocalendar)
+
+add_test(tst_calendarevent tst_calendarevent)
```

Comment:
Like in `sailfishos/mkcal`, tests should be installed, for instance with:

```
if(INSTALL_TESTS)
	install(TARGETS tst_calendarevent
		DESTINATION /opt/tests/nemo-qml-plugins-qt5/calendar)
endif()
```

see old `common.pri`.

Reply:
I never understood this, why exactly? I have no problem adding it under a conditional block (so at least I have the option to not install them) but I'd like to understand why you install them in the first place.

Reply:
Sorry, I'm not working at Jolla. I'm part of the community also. Maybe @pvuorela could answer.

I guess that they want RPMs to easily deploy tests on a test infrastructure. Now, this does not necessary imply to put the installation procedure in the build system itself of the project. It could be done at the spec level for specific SailfishOS builds. I agree. But well, I guess it's historical and maybe easier also than writing `install -m 644 -p...` multiple times in the spec file.

Reply:
We do automatic nightly run for the unit tests and the setup requires that there's the tests.xml and the test binaries installed.